.gitignore

```diff
@@ -6,3 +6,4 @@ hc/local_settings.py
 static-collected
 
 .DS_Store
+hc/api/tests/.cache/v/cache/lastfailed
```

Comment:
I think we can target directory `.cache` so that in case another file is autogenerated in a different tests directory we won't have to ignore it too